### PR TITLE
Html text strings

### PR DESCRIPTION
**Pull request by SychO9**
If the devs approve of wrapping the HTML in source files, feel free to merge this to your html-text-string branch, when you do so these commits will show up in your pull request, you can edit things however you want, just remember to pull the changes to your local branch before.

### CODE REVIEW

File: Sources/ManageErrors.php [hunk 1]
```diff
@@ -44,6 +44,9 @@ function ViewErrorLog()
 	loadLanguage('ManageMaintenance');
 	loadTemplate('Errors');
 
+	// Wrapping HTML around text strings
+	$txt['errortype_critical'] = '<span class="red">' . $txt['errortype_critical'] . '</span>';
```
Comment (MissAllSunday) [inline Sources/ManageErrors.php]: oh nope!

This is basically moving the html from txt files to source ones. We can indeed place some HTML on source files if and only if they cannot be moved to template files or if it requires too much changes to be able to do so.

Each case will be different and has to be thought out separately. If its too much trouble then leave it as it is ;)

File: Sources/ManageServer.php [hunk 2]
```diff
@@ -770,13 +770,13 @@ function ModifyLoadBalancingSettings($return_config = false)
 	{
 		$context['settings_message'] = $txt['loadavg_disabled_windows'];
```
Comment (MissAllSunday) [inline Sources/ManageServer.php]: Where in the template does $context['settings_message'] gets printed? is it a reusable context var?

File: Sources/ManageServer.php [hunk 3]
```diff
@@ -795,6 +795,8 @@ function ModifyLoadBalancingSettings($return_config = false)
 		}
 	}
 
+	$context['settings_message'] = '<span class="error">' . $context['settings_message'] . '</span>';
```
Comment (MissAllSunday) [inline Sources/ManageServer.php]: It seems it is a reusable var.  In this case why don't we turn it into an array? something like:

`
$context['settings_message'] = array(
	'message' => $some_txt_var,
	'tag' => 'strong|span|div|whatever',
	'css_class' => 'alert|error',
);
`

Its not pretty but at least we decompose the tags and added a way to print multiple tags using the same $context var.

File: Sources/Modlog.php [hunk 4]
```diff
@@ -435,7 +435,7 @@ function list_getModLogEntries($start, $items_per_page, $sort, $query_string = '
 		{
 			// Guests don't have names!
 			if (empty($row['extra']['member']))
-				$row['extra']['member'] = $txt['modlog_parameter_guest'];
+				$row['extra']['member'] = '<em>' . $txt['modlog_parameter_guest'] . '</em>';
```
Comment (MissAllSunday) [inline Sources/Modlog.php]: Haven't check but this doesn't seem like something difficult to move the HTML to a template or is it? It looks like the standard while fetch loop to put data from the DB into a context var.

File: Sources/Who.php [hunk 5]
```diff
@@ -42,6 +42,10 @@ function Who()
 	loadTemplate('Who');
 	loadLanguage('Who');
 
+	// Wrapping HTML around text strings
```
Comment (MissAllSunday) [inline Sources/Who.php]: Again, nope! lets try moving the HTML to the template, if it cannot be done then lets find a way to decouple the tag from the text.

File: Sources/Modlog.php (hunk 4 above)
Comment (SychO9) [reply Sources/Modlog.php]: It uses the `createList()` function which uses a generic template used in most admin pages, so I'm not sure it's doable

File: Sources/ManageServer.php (hunk 3 above)
Comment (SychO9) [reply Sources/ManageServer.php]: Done, cleaned up more HTML in other source files by doing so

File: Sources/Who.php (hunk 5 above)
Comment (SychO9) [reply Sources/Who.php]: Done, used `is_array()` check to try and maintain backwards compatibility with mods, let me know if you'd rather not have that

File: Sources/Modlog.php (hunk 4 above)
Comment (frandominguez03) [reply Sources/Modlog.php]: Can you remove this changes? I'm looking to merge this and push to the original PR.

**Pull request merged**